cookbook/configuration/web_server_configuration.rst

```diff
@@ -25,14 +25,16 @@ to use PHP :ref:`with Nginx <web-server-nginx>`.
     web server. In the examples below, the ``web/`` directory will be the
     document root. This directory is ``/var/www/project/web/``.
 
+    If your hosting provider requires you to change the ``web/`` directory to
+    another location (e.g. ``public_html/``) make sure you
+    :ref:`override the location of the web/ directory <override-web-directory>`.
+
 .. _web-server-apache-mod-php:
 
 Apache2 with mod_php/PHP-CGI
 ----------------------------
 
-For advanced Apache configuration options, see the official `Apache`_
-documentation. The minimum basics to get your application running under Apache2
-are:
+The **minimum configuration** to get your application running under Apache2 is:
```

Comment:
You are right. I've removed the `2` suffix even for the section titles. Do you think is safe to do that considering that they are just subsections instead of page titles? Thanks.